Assets/MixedRealityToolkit-Examples/Demos/Boundary/Scripts/BoundaryVisualizationDemo.cs

```diff
@@ -136,12 +133,12 @@ private void AddMarkers()
 
                     Material material = null;
                     // Check inscribed rectangle first
-                    if (BoundaryManager.Contains(position, Boundary.Type.PlayArea))
+                    if (MixedRealityManager.BoundarySystem.Contains(position, Boundary.Type.PlayArea))
```

Comment:
Not checking for no boundary system
```
MixedRealityManager.BoundarySystem?.
```

Reply:
We check for null before we `AddMarkers`